index.html

```diff
@@ -1,18 +1,64 @@
-<script>
-let name = document.location.pathname.slice(1);
+<!DOCTYPE html>
+<html lang="en">
 
-if (!name) {
-  console.log("no place name specfied");
-} else {
-  console.log("fetching place named '" + name + "'");
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="style.css" rel="stylesheet" />
+  <title>Document</title>
+  <link rel="stylesheet" href="https://unpkg.com/leaflet@1.6.0/dist/leaflet.css"
+    integrity="sha512-xwE/Az9zrjBIphAcBb3F6JVqxf46+CDLwfLMHloNu6KEQCAWi6HcDUbeOfBIptF7tcCzusKFjFw2yuvEpDL9wQ=="
+    crossorigin="" />
+  <script src="https://unpkg.com/leaflet@1.6.0/dist/leaflet.js"
+    integrity="sha512-gZwIG9x3wUXg2hdXF6+rVkLF/0Vi9U8D2Ntg4Ga5I5BZpVkVxlJWbSQtXPSiUTtC0TjtGOmxa1AJPuV0CPthew=="
+    crossorigin=""></script>
+</head>
 
-  fetch(name + '.json')
-    .then(function (response) {
-      return response.text();
-    })
-    .then(function (myText) {
-      console.log(myText);
-    });
-}
+<body>
+  <!--Title Header-->
+  <section class="main-container">
+    <!--Title Container-->
+    <section class="title-flex-container">
+      <p id="yelpington-flex"><a href="http://localhost:5000/" style="text-decoration: none">YELPINGTON...</a></p>
```

Comment:
Here, you would want your href to be ```"/"``` or ```"index.html"``` because if this was running on a hosting service, somewhere like heroku, this would look for localhost:5000 in the service the app is running in, which isn't the index page that you actually want to go to. This works here because when you run your program you are always starting it on your computer which is localhost and it always starts on port 5000. This may not be the case when your program is started elsewhere.